### ISSUE TITLE
Interactive Docs

### ISSUE THREAD

**Issue by AbdulTheActivePiecer** [Airtable/airtable.js#349]
I can't find the docs for this package, can someone refer the way to find them ? the way mention in the readme is incorrect, or at least I can't find it.

**Comment by sklnd** [comment]
@AbdulTheActivePiecer I found that the interactive documentation is available by selecting a specific base under `To view API documentation that is generated for a particular base, see below:`. The documentation there is updated dynamically for the selected base. It is a bit confusing that both links in the README direct to the same docs, but the dynamic documentation is pretty nice.

**Comment by reustle** [comment]
While this documentation helps for Base related info, it seems to be lacking in the general usage / authentication side.

For example, I know I can pass in my API key, but what about refresh tokens? Will it handle that for me, or do I need to make manual calls to the API myself? It seems like I'll need to read through the code to find out, and would be happy to help w/ the docs here if I had a better understanding.

**Comment by EndBug** [comment]
Yeah, I'm not able to find any docs on how to use this library: TypeScript types are incomplete, and the online docs only refer to the API endpoints, not to this package...

While I get that there might be more info available if you log in, this means that open source contributors just have to either guess or look at the source to figure out how they can use the package :/
